### DISCUSSION TITLE
Why do I get this error when installing Copilot 1.3.1 from the Marketplace?

### DISCUSSION THREAD

**Discussion by crojasdxc** [Genuitec/Copilot4Eclipse-community#43] [Help] [answered]
![image](https://github.com/user-attachments/assets/f787904d-5d41-461b-b4a8-04b74e741944)

**Comment by wayneparrott** [comment]
From your screenshot, did you "See the error log for details." as suggested?

After you review the error log details if you still require assistance, share as much info as you have available. Follow this guide for posting and we can probably fix this up quickly: https://github.com/Genuitec/Copilot4Eclipse-community/discussions/2 

Wayne
Copilot4Eclipse Team

**Reply by crojasdxc** [reply]
Good morning, I am reporting the error that occurred when consulting the URL for the option to install new software.

the message for both errors is the same

<img width="395" alt="image" src="https://github.com/user-attachments/assets/9ef8133f-991a-4bd7-97e4-2809a00079a2">

**Comment by wayneparrott** [comment]
0. What Eclipse version and distro are you running?

1. What version of Java and distro are you running Eclipse with?
 
2. Are you operating from behind a corporate proxy? 

3. In order to isolate the problem please install a plugin other than Copilot4Eclipse and not a core Eclipse plugin from the Marketplace. Here's a popular 3rd party plugin to install: https://marketplace.eclipse.org/content/darkest-dark-theme-devstyle 
Share the result. This test is to help rule out the possibility that your network is locked down preventing any plugin from being installed.

W

**Reply by crojasdxc** [reply, marked as answer]
Thank you for your support, it was identified that it was a proxy block. 

this was the solution

Solution-1: Define genuitec.com as a trusted site
The simplest solution is to trust the *.genuitec.com domains and omit them from the network proxy's SSL inspection. This usually involves requesting your network support team add the *.genuitec.com domains to the network proxy's list of trusted sites. Update the proxy configuration is typically a quick low complexity task.

https://www.genuitec.com/products/copilot4eclipse/docs/resources/technotes/installation-pkix-fix/
